app/routes/unit_tests/test_projects.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+from flask import jsonify, current_app
+from unittest.mock import patch
+from mock_data import MOCK_DATA
+
+class status:
+    HTTP_201_CREATED = 201
+    HTTP_200_OK = 200
+    HTTP_400_BAD_REQUEST = 400
+    HTTP_204_NO_CONTENT = 204
+
+from flytrap import app
+app.config["TESTING"] = True
+
+@pytest.fixture
+def client():
+    return app.test_client()
+
+@patch('app.routes.projects.add_project')
+@patch('app.routes.projects.generate_uuid')
+def test_create_project(mock_generate_uuid, mock_add_project, client):
+    mock_generate_uuid.return_value = "dajhew32876dcx79sd2332"
+    mock_add_project.return_value = MOCK_DATA["create_project"]
+
+    response = client.post('/api/projects', json={
+        "name": "testing123",
+        "platform": "react"
+    })
+
+    response_json = response.get_json()
+
```

Comment:
I think this test would currently only fail if  line 108 was commented out in the actual route. Could there be some sort of test that the model is at least called with the right parameters?
Also, what about the AWS calls? Do they need to be mocked?

Reply:
Yes, the next step will be to add an assertion to test that the mocked model is called with the right arguments. The AWS service calls are currently not being made because we are using flask's app.test_client object to make HTTP requests. However, it would be better to explicitly mock AWS service calls for consistency and to test that they are being made with the correct arguments. This will be the next phase of work.